cdap-ui/app/cdap/components/DataPrep/ColumnActionsDropdown/ColumnActionsDropdown.scss

```diff
@@ -58,6 +63,7 @@
       font-weight: 400;
       line-height: 2;
       cursor: initial;
+      box-shadow: 1px 1px #cccccc;
```

Comment:
@ajainarayanan Already added the box-shadow on the main menu, but accidentally applied it to the wrong location... It's the `box-shadow: 1px 1px 5px #cccccc;` that was added to `.popover-content` in the `DataPrepSidePanel`, will move it to the `.popover-content` over here.

Once that's done then the second level pop-over will have the same depth, so maybe we don't need to add it?